doc/authentication-methods/client-certificate.md

```diff
@@ -11,6 +11,17 @@ A server must request the certificate from a client, so you'll need to enable `v
 
 Please check [Listener modules](../advanced-configuration/Listener-modules.md#client-to-server-c2s-ejabberd_c2s) page for more information or simply follow the examples at the end of this section.
 
+### Properly configure `ejabberd_cowboy` listener
+
+SASL EXTERNAL authentication is also possible for WebSocketSecure and BOSH connections over HTTPS.
+Similarly to `ejabberd_c2s` case, the server must request the certificate from the client.
+In this case it's enabled by adding following options to `ssl` option of `ejabberd_cowboy` :
+
+* `{verify, verify_peer}` - this is to tell Erlang's SSL to request the cert from the client
+* `{cacertfile, "/path/to/ca.pem"}` - this is to tel Erlang's SSL where is the CA cert file in order to check if the cert is correctly signed
```

Comment:
{cacertfile, "/path/to/ca.pem"}` - this is to tell Erlang's SSL where the CA cert file is in order to check if the cert is correctly signed